drivers/wifi/nrf700x/zephyr/src/zephyr_wpa_supp_if.c

```diff
@@ -1152,3 +1152,170 @@ int wifi_nrf_nl80211_send_mlme(void *if_priv, const u8 *data,
 
 	return status;
 }
+
+struct wpa_supp_event_supported_band
```

Comment:
Did you mean to return a struct or a pointer to a struct? Better to take pointer to band as one more input parameter and return success/fail as return value.